### PR TITLE
Kái sán-phín miâ, lō͘-kèng, siat-tēng téng-téng. Kā plum the̍h-tiāu. Siu-kái AppVeyor script, and add GitHub Release.

### PR DESCRIPTION
**Pull request by ngoohebi**

### CODE REVIEW

File: WeaselTSF/WeaselTSF.vcxproj [hunk 1]
```diff
@@ -119,35 +119,35 @@
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|Win32'">
     <LinkIncremental>true</LinkIncremental>
     <OutDir>$(SolutionDir)output\</OutDir>
-    <TargetName>weasel</TargetName>
+    <TargetName>ThuanTaigi</TargetName>
```
Comment (ngoohebi) [inline WeaselTSF/WeaselTSF.vcxproj:122]: @sih4sing5hong5  Chia khai-sí sī WeaselTSF lāi-té dll output kái-miâ ê pō͘-hūn,

File: WeaselIME/WeaselIME.vcxproj [hunk 2]
```diff
@@ -121,13 +121,13 @@
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|Win32'">
     <OutDir>$(SolutionDir)output\</OutDir>
     <LinkIncremental>false</LinkIncremental>
-    <TargetName>weasel</TargetName>
+    <TargetName>ThuanTaigi</TargetName>
```
Comment (ngoohebi) [inline WeaselIME/WeaselIME.vcxproj:124]: @sih4sing5hong5  Chia khai-sí sī WeaselIME lāi-té dll output kái-miâ ê pō͘-hūn,

File: output/install.nsi [hunk 3]
```diff
@@ -127,21 +128,21 @@ program_files:
   File "curl-ca-bundle.crt"
   File "rime-install.bat"
   File "rime-install-config.bat"
-  File "weasel.dll"
+  File "ThuanTaigi.dll"
```
Comment (ngoohebi) [inline output/install.nsi:131]: @sih4sing5hong5  Chia khai-sí sī dll output an-chong ê tóng-àn.

**Comment by ngoohebi** [comment]
Oân-sêng:
1. Kái sán-phín miâ, lō͘-kèng, siat-tēng téng-téng. 
2. Kā plum the̍h-tiāu. 
3. Siu-kái AppVeyor script, and add GitHub Release.

**Comment by ngoohebi** [comment]
@sih4sing5hong5 Chhiáⁿ merge. ló͘-la̍t.

**Comment by ngoohebi** [comment]
Oân-sêng:

Kái sán-phín miâ, lō͘-kèng, siat-tēng téng-téng.
Kā plum the̍h-tiāu.
Siu-kái AppVeyor script, and add GitHub Release.
Release tóng-àn-miâ cheng-ka pán-pún kap sî-kan phiau-kì.

Ē-sái merge.
@sih4sing5hong5

**Review by sih4sing5hong5 — APPROVED** [review]

**Pull request merged**